packages/docusaurus-types/package.json

```diff
@@ -16,7 +16,9 @@
     "test": "tsc -p ."
   },
   "dependencies": {
+    "@docusaurus/react-loadable": "5.5.2",
```

Comment:
I'm not entirely sure about our package alias setup. PnP is able to resolve this?

Reply:
Good catch, you're right. Just ran into this when attempting to work around this in my own project - my initial test was incorrect. Retested

Reply:
Actually... you just need to change the code and import the type from `@docusaurus/react-loadable` instead.

Reply:
Ah, that actually doesn't work. `react-loadable`'s type is weird.